app/controllers/idv/phone_errors_controller.rb

```diff
@@ -21,12 +21,12 @@ def warning
     end
 
     def timeout
-      @remaining_step_attempts = rate_limiter.remaining_count
+      @remaining_submit_attempts = rate_limiter.remaining_count
```

Comment:
The controller did have a method that was `remaining_step`, but as it was defined the exact same way as others in this file named `remaining_attempt`, I changed it to be consistent.

Reply:
@night-jellyfish , can you point to where is the method?

Reply:
That's a good point. It doesn't seem like it's ever called, outside of [the spec files](https://github.com/18F/identity-idp/pull/10011/files#diff-5b425adf156d49e3b6c10018750f4922dbd50f7ae5b130ccf80819c5062fee66R187). Perhaps it can just be removed then?

Reply:
@night-jellyfish , i think like `job_fail`, the reading of `rate_limiter.remaining_count` may have side effects that needed, like update the expiry time of underlying redis_key, otherwise the `RateLimiter` will not be refreshed though these urls are visisted.

That raises another question why we creating a new `RateLimiter` instance every time here? It works since we only use it once, and also it's backed by `Redis` by the same key.

Anyway, interesting details.

Reply:
@night-jellyfish , i think it's fine to leave it as is, for `Redis`, just treat it as a database.

Reply:
I see. Just for my own understanding then - are you saying it'd be better to re-use the same RateLimiter instance because we could just reuse a single row in the Redis DB vs creating a new row each time?